pandas/core/series.py

```diff
@@ -2502,6 +2502,15 @@ def _binop(self, other, func, level=None, fill_value=None):
         -------
         Series
         """
+
+        def mk_ret(result, new_index, name):
```

Comment:
Can you reuse anything from `core/ops.py` here? In particular, `_construct_divmod_result`.